IBAnimatable/ActivityIndicatorAnimationBallBeat.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created by Tom Baranes on 21/08/16.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+public class ActivityIndicatorAnimationBallBeat: ActivityIndicatorAnimating {
+
+  // MARK: Properties
+
+  private let duration: CFTimeInterval = 0.7
+
+  // MARK: ActivityIndicatorAnimating
+
+  public func configAnimation(in layer: CALayer, size: CGSize, color: UIColor) {
+    let circleSpacing: CGFloat = 2
+    let circleSize = (size.width - circleSpacing * 2) / 3
+    let x = (layer.bounds.size.width - size.width) / 2
+    let y = (layer.bounds.size.height - circleSize) / 2
+    let beginTime = CACurrentMediaTime()
+    let beginTimes = [0.35, 0, 0.35]
+
+    // Draw circles
+    for i in 0 ..< 3 {
+      let circle = ActivityIndicatorShape.Circle.createLayerWith(size: CGSize(width: circleSize, height: circleSize), color: color)
+      let frame = CGRect(x: x + circleSize * CGFloat(i) + circleSpacing * CGFloat(i),
+                         y: y,
+                         width: circleSize,
+                         height: circleSize)
+      animation.beginTime = beginTime + beginTimes[i]
+      circle.frame = frame
+      circle.addAnimation(animation, forKey: "animation")
+      layer.addSublayer(circle)
+    }
+  }
+
+}
+
+// MARK: - Setup
+
+private extension ActivityIndicatorAnimationBallBeat {
+
+  var scaleAnimation: CAKeyframeAnimation {
```

Comment:
In the future, if those properties can be reused in other classes confirming to `ActivityIndicatorAnimating`, we can make them in `ActivityIndicatorAnimating` protocol extension.